### PR TITLE
website(docs): Adds a link to contributor page from the latest documentation page

### PR DESCRIPTION
**Pull request by PaulRMellor**
Signed-off-by: prmellor <pmellor@redhat.com>

### Type of change

_Select the type of your PR_

* [x] Typo/minor fix
* [ ] New blog post (see the [README](https://github.com/strimzi/strimzi.github.io/#blog-posts) for the process)
* [ ] Other

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
TBH, I prefer the original design. It aligns ad wraps better, the title added it a good spacing for the section, the columns are equally long etc.

This Pr makes it all cramped up and mashed together and quite hard to find the actual books etc.

**Comment by PaulRMellor** [comment]
> TBH, I prefer the original design. It aligns ad wraps better, the title added it a good spacing for the section, the columns are equally long etc.
> 
> This Pr makes it all cramped up and mashed together and quite hard to find the actual books etc.

Now I can the see preview, I can do something with it. I wasn't entirely sure how it might look from the changes locally. The contribute bit was meant to span the whole length.

**Review by scholzj — COMMENTED** [review]
Do we really need so many sub-headers? We have the main header. And then on the left side, we have:
* `STRIMZI 0.32.0 DOCUMENTATION`
* `STRIMZI 0.32.0 KAFKA OPERATORS`
* `STRIMZI KAFKA BRIDGE 0.22.3`

I would remove the `STRIMZI 0.32.0 DOCUMENTATION`. Also, do we need the version everywhere? I would either remove it from the subheaders (i.e. keep only `STRIMZI KAFKA OPERATORS` for example). Or I would remove it from the links.

----

On the right side, I would probably keep the original layout:
* It better explains what it is
* It makes the links to the in-development docs less pronounced. The in-development docs is basically _for experts_ -> for people who know that they are looking for it. But in the design of this PR, it is basically put at level with the 0.32 docs and IMHO users will be confused which one to use. 

----

The `CONTRIBUTE TO THE DOCUMENTATION` section looks good.

File: _includes/documentation/documentation-version-band.html [hunk 1]
```diff
@@ -1,39 +1,56 @@
...
-        Configuring Strimzi: <a href="{{site.baseurl}}/docs/operators/latest/configuring.html">{{site.data.releases.operator[0].version}} - latest stable release</a> <a href="{{site.baseurl}}/docs/operators/latest/full/configuring.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a><br/>
-        Example custom resources: <a href="{{site.github_url}}/strimzi-kafka-operator/tree/{{site.data.releases.operator[0].version}}/examples">{{site.data.releases.operator[0].version}} - latest stable release</a>
-      </p><br/>
+        <a href="{{site.baseurl}}/docs/operators/latest/overview.html">Overview ({{site.data.releases.operator[0].version}})</a> <a href="/docs/operators/latest/full/overview.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> <br/>
+        <a href="{{site.baseurl}}/docs/operators/latest/quickstart.html">Quick Start guide ({{site.data.releases.operator[0].version}})</a> <a href="/docs/operators/latest/full/quickstart.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> <br/>
```
Comment (scholzj) [inline _includes/documentation/documentation-version-band.html:14]: Any reason why this one has the `guide` and others don't?

Comment (PaulRMellor) [reply _includes/documentation/documentation-version-band.html:14]: I'll remove that. Don't think we need it

**Comment by PaulRMellor** [comment]
@scholzj - I've reverted the in dev doc to the original format. I preferred the more symmetrical layout on a visual level, but at least it's a bit cleaner than the original now. I was more than happy to take out the version numbers from the links. I assumed we wanted to retain them

**Review by scholzj — APPROVED** [review]

**Pull request merged**